app/tests/e2e/interface.spec.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
     await browser.waitForElementVisible(".li-validator", 10000)
     const validatorNames = await getValidators(browser)
 
+    await browser.click(".toggles .btn-radio:nth-child(3)")
```

Comment:
Woah, good catch. Didn't think of the implications for E2E on the popularity one. Probably neither of us paid attention there to E2E (monorepo was so new)